### PR TITLE
Docs for StashCache ready to publish

### PR DESCRIPTION
**Pull request by zvada**
Quite substantial edits to merge. Included table of content in mkdocs.yml. I think is in fair condition to publish for someone to install stuff on RHEL7. For RHEL6 it needs few more edits, but easy to figure even without it for experienced admin.

### CODE REVIEW

File: docs/admin/configure-cache.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+The following section describes required configuration to have a functional StashCache Cache (not origin server!). StashCache Cache package `stashcache-cache-server` needs to be manually configured from pre-existing XRootD configuration.
+
+## Cache server
+!!! Note: 
+:bangbang: While example of the configuration file below provides combination of _authenticated_ and _non-authenticated_ _Cache_, the non-authenticated cache config is considered to be default option. If you're about to configure in addition _authenticated cache_ read to the end of this document and then follow post-installation of [authenticated part here](configure-cache-auth.md).
+
+For configuring **cache** one needs to define directive `pss.origin redirector.osgstorage.org:1024` (not `all.manager redirector.osgstorage.org+ 1213` directive as it is in case of [configuring origin](configure-origin.md)). 
+`StashCache-daemon` package provides default configuration file `/etc/xrootd/xrootd-stashcache-cache-server.cfg`. Example of the configuration of cache server is as follows:
+```
```
Comment (djw8605) [inline docs/admin/configure-cache.md]: I thought you were going to use the include features for this?

File: docs/admin/configure-cache-auth.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+2. Reload daemons:
+```
+   [root@client ~]$ systemctl daemon-reload
+```
+
+#### Proxy.service
+1. Create the file with following content:
+```
+   [root@client ~]$ cat /usr/lib/systemd/system/xrootd-renew-proxy.service
```
Comment (djw8605) [inline docs/admin/configure-cache-auth.md]: I don't see the place where you create the `override.conf` file to specify the proxy to the auth-stashcache

File: docs/admin/configure-cache.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+### Add Robots file
+```
+   [root@client ~]$ cat /etc/xrootd/stashcache-robots.txt 
+   User-agent: *
+   Disallow: /
+```
+
+### RHEL7
+On RHEL7 system, you need to run following systemd unit:
+* `xrootd@stashcache-cache-server.service`
```
Comment (djw8605) [inline docs/admin/configure-cache.md]: Why not give the full command, `systemctl start xrootd@stashcache-cache-server.service`

File: docs/admin/configure.md [hunk 4]
```diff
@@ -112,24 +115,126 @@ xrootd.monitor all auth flush 30s window 5s fstat 60 lfn ops xfr 5 dest redir fs
...
 
+# Optional configuration
 # Remote debugging
 xrootd.diglib * /etc/xrootd/digauth.cf
 ```
 
-### Enabling authenticated cache
+#### Add AuthFile for non-authenticated cache
+```
+   [root@client ~]$ cat /etc/xrootd/Authfile-noauth
```
Comment (djw8605) [inline docs/admin/configure.md]: Isn't this already in the configure-cache-auth file?

File: docs/admin/install.md [hunk 5]
```diff
@@ -1,15 +1,30 @@
 # Installation Guide
 
-This document describes how to install a StashCache cache, a server that caches files locally rather than serving files to the StashCache federation. The installation utilizes XRootD and HTCondor for file storage and monitoring, respectively.
+This document describes how to install a StashCache service (Origin or Cache). The installation utilizes XRootD and HTCondor for file storage and monitoring, respectively.
```
Comment (djw8605) [inline docs/admin/install.md]: Can you describe the role of the Origin and Cache?  When would you need to install which?

File: docs/index.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
 # StashCache
 
-This documenation serve to Operations team and Site Admins who manage StashCache nodes (origin or cache server; or redirectors). Main functionality of StashCache is based on [XRootD](http://xrootd.org) technology, there are other components involved to function StashCache properly, though. E.g. if you are a user of **"stashcp"** you want to follow [this](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcache) document instead.
+This documenation serves to Operations team and Site Admins who manage StashCache nodes (origin or cache server; or redirectors). Main functionality of StashCache is based on [XRootD](http://xrootd.org) technology, there are other components involved to function StashCache properly, though. E.g. if you are a user of **"stashcp"** you want to follow [this](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcache) document instead.
```
Comment (djw8605) [inline docs/index.md]: > This documentation is for Side Admins and OSG Operations team who manage...

File: docs/index.md [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
-* [StashCache caches](https://github.com/opensciencegrid/StashCache/docs/admin): Managed by an organization, **"cache"** server to keep data cached and immediately available (via [stashcp](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcach))within Stash federation (without re-transfering from "origin").
-* [StashCache upgrades](https://twiki.grid.iu.edu/bin/view/Documentation/Release3/StashCacheUpgrades): Keeps track of versions deployed at sites as well as other information about OS, system resources, connectivity, etc.
+* [StashCache origins](admin/install.md): Managed by an organization, **"origin"** is data server to host files locally and serve them to users upon transfer request (via [stashcp](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcach)) unless data are already cached.
```
Comment (djw8605) [inline docs/index.md]: You should reorder this list.  Caches at top (most common).  Then origins, redirectors, and upgrades.

File: docs/ops/start.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+| **Software** | **Service name** | **Notes** |
+|--------------|------------------|-----------|
+| XRootD | `xrootd@stashcache-cache-server-auth.service` | RHEL7 |
+|  | `xrootd-renew-proxy.service` | RHEL7 |
+|  | `xrootd-renew-proxy.timer` | RHEL7  |
+| HTCondor | `condor.service` | RHEL7  |
+| Fetch CRL | `fetch-crl-cron` | RHEL7 |
+
+### Validating Cache server
+To verify that your cache is being monitored properly, run the following command:
```
Comment (djw8605) [inline docs/ops/start.md]: There has to be more of a test than just this.  You should include a `wget` command to test the CVMFS accessibility.

    curl -O http://cache_host:8000/user/dweitzel/public/blast/queries/query1

Or something like that.  I'm sure there is a curl command that doesn't produce output.

**Comment by bbockelm** [comment]
There are lots of commits to this branch with no useful commit messages.

Accordingly, when this is merged, please rebase + squash with a useful commit message.

**Pull request merged**